mne/preprocessing/ica.py

```diff
@@ -548,8 +548,7 @@ def _transform_epochs(self, epochs, concatenate):
         if not hasattr(self, 'mixing_matrix_'):
             raise RuntimeError('No fit available. Please fit ICA')
 
-        picks = pick_types(epochs.info, include=self.ch_names, exclude=[],
-                           ref_meg=False)
+        picks = [epochs.ch_names.index(k) for k in self.ch_names]
```

Comment:
It's weird that you have to resort to using a list comprehension, I'd assume that `pick_types` or `pick_channels` would do what you wanted